data/ships.txt

```diff
@@ -1480,6 +1480,46 @@ ship "Gunboat"
 
 
 
+ship "Harvester"
+	sprite "ship/mining drone"
+	attributes
+		category "Drone"
+		"cost" 100000
```

Comment:
That's EXPENSIVE for a human drone. I'd put this to at most 80k, just under the combat drone's hull cost. But make sure it stays above 50k, the surveillance drone's hull cost.